src/containers/ViewerPanel/test/selectors.test.ts

```diff
@@ -70,7 +74,95 @@ describe("ViewerPanel selectors", () => {
                 roundedTime: 6,
                 roundedLastFrameTime: 30,
                 roundedTimeStep: 0.2,
+                maxNumChars: 4,
             });
         });
+
+        it("shows correct maxNumChars when firstFrameTime is very long", () => {
+            const mockState: State = {
+                ...initialState,
+                selection: {
+                    ...initialState.selection,
+                    time: 3.0001,
+                },
+                metadata: {
+                    ...initialState.metadata,
+                    timeUnits: {
+                        magnitude: 1,
+                        name: "ns",
+                    },
+                    timeStep: 0.0800000000023,
+                    lastFrameTime: 15.0001,
+                    firstFrameTime: 0.000004000000001,
+                },
+            };
+
+            const displayTimes = getDisplayTimes(mockState);
+
+            expect(displayTimes).to.deep.equal({
+                roundedTime: 3,
+                roundedLastFrameTime: 15,
+                roundedTimeStep: 0.08,
+                maxNumChars: 8,
```

Comment:
i'm not sure i understand why this is 8 since the rounded times are all to 2-3 sig figures.

Reply:
Ah, yeah it's because `firstFrameTime` is 0.000004. I guess I could do this instead like you suggested in another comment:
```
            expect(displayTimes).to.deep.equal({
                roundedTime: 3,
                roundedLastFrameTime: 15,
                roundedTimeStep: 0.08,
                maxNumChars: "0.000004".length(),
            })
```

Reply:
ah, i also can't quite count :)